vlib/v/checker/tests/method_op_alias_err.out

```diff
@@ -0,0 +1,28 @@
+vlib/v/checker/tests/method_op_alias_err.vv:4:1: error: both sides of an operator must be the same type
```

Comment:
more like
```
    4 | fn (f Foo) + (f1 Foo2) Foo2 {
      |                  ~~~~
```
paramater has to be the same type like the receive on overloading.

Reply:
Hmm...  I would use

`operation only supported on similar types`

and I would use that on all operations, no matter the type.  "similar" instead of "same" because regular `+` should work without a problem on `1 + 2.0` - even though they are different types, they are still "similar" (as they are both numbers).

For `+=` the result just has to be castable to the receiver.

Reply:
Yes